server/meshmodel/kubernetes/relationships/hierarchical_secret.json

```diff
@@ -0,0 +1,172 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+      "description": "Use this relationship to make sensitive information available to other resources, like your application, by environment variable.",
+      "styles": {}
+  },
+  "model": {
+      "name": "kubernetes",
+      "version": "v1.25.2",
+      "category": {
+          "name": "Orchestration \\u0026 Management",
+          "metadata": null
+      },
+      "metadata": {}
+  },
+  "subType": "Secret",
```

Comment:
@MUzairS15, this need to be `Secret` temporarily until you've added `SelectionSets` (or arrays of selections), right?